.github/workflows/java-ci.yml

```diff
@@ -95,7 +95,7 @@ jobs:
     runs-on: ubuntu-22.04
     strategy:
       matrix:
-        jvm: [11, 17, 21]
+        jvm: [17, 21]
```

Comment:
This is for `build-checks`. Not able to do `build-checks` using java 11 because Spark 4.0 is build using java 17.